test/fixtures/auth-requests.json

```diff
@@ -198,7 +199,8 @@
               "saveHelperData": true,
               "extraData": "skjdfklsjhdflkjhsdf",
               "app": "someAppId",
-              "delegation": ""
+              "delegation": "",
+              "includePayloadHash": true
```

Comment:
If echo has no effect based on payload hash, it's better to have a local server which respect/throw-error based on the presence of hash.

Reply:
Echo has support for payload verification now.